project/Dependencies.scala

```diff
@@ -147,6 +148,7 @@ object Dependencies {
   private val sttpClient = sttpClientModule("core")
   private val sttpPlayJson = sttpClientModule("play-json")
   private val sttpAsyncHttpClient = sttpClientModule("async-http-client-backend-future")
+  private val asyncHttpClient = "org.asynchttpclient" % "async-http-client" % Version.asyncHttpClient
```

Comment:
Why do we need this? This dependency retrieved transitively through `async-http-client-backend-future`, so we need to update `sttpClient` version, e.g. to 3.2.3: https://github.com/softwaremill/sttp/blob/v3.2.3/build.sbt

Reply:
resolved